frontend/amundsen_application/static/js/ducks/log/api/v0.ts

```diff
@@ -14,6 +14,7 @@ export interface ClickLogParams {
   target_type?: string;
   label?: string;
   value?: string;
+  position?: string;
```

Comment:
Canonically we've been using `index` instead of position. Might consider changing to be consistent.